### PR TITLE
 Add Deprecations and Removals for Chrome 76.

### PR DESCRIPTION
**Pull request by jpmedley**
**Target Live Date:** 2019-06-13

- [ ] This has been reviewed and approved by (NAME)
- [ ] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{# wf_updated_on: 2019-06-13 #}
+{# wf_published_on: 2019-06-13 #}
+{# wf_tags: deprecations,removals,chrome76 #}
+{# wf_blink_components: Blink #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 76 to help you plan. #}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
+
+# Deprecations and removals in Chrome 75 {: .page-title }
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
# Deprecations and removals in Chrome 76 {: .page-title }
```

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+## Remove Feature Policy: lazyload
+
+The lazyload feature policy was intended to allow developers to selectively
+control the lazyload attribute on `&lt;iframe>` and `&lt;img>` to provide more
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
control the `lazyload` attribute on `<iframe>` and `<img>` to provide more
```

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+control over loading delay for embedded contents and images on a per origin
+basis. 
+
+The policy is removed in favor of a newer feature policy for loading, namely
+loading-frame-default-eager which is more aligned with how the "loading"
+attribute will be used. The removal applies to both the Feature-Policy header
+and the `&lt;iframe>` `allow` attribute.
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
and the `<iframe>` `allow` attribute.
```

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+## Remove Feature Policy: lazyload
+
+The lazyload feature policy was intended to allow developers to selectively
+control the lazyload attribute on `&lt;iframe>` and `&lt;img>` to provide more
+control over loading delay for embedded contents and images on a per origin
+basis. 
+
+The policy is removed in favor of a newer feature policy for loading, namely
+loading-frame-default-eager which is more aligned with how the "loading"
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
`loading-frame-default-eager` which is more aligned with how the `loading`
```

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+attribute will be used. The removal applies to both the Feature-Policy header
+and the `&lt;iframe>` `allow` attribute.
+
+[Chrome Platform Status](https://www.chromestatus.com/feature/5641405942726656) &#124;
+[Chromium Bug](https://crbug.com/869492)
+
+
+## Remove Insecure Usage of DeviceMotionEvent
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
## Remove Insecure Usage of `DeviceMotionEvent`
```

File: src/content/en/updates/2019/06/chrome-76-deps-rems.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+used in a non-secure browsing context. Chrome now restricts the API to secure
+browsing contexts. This change brings Chromium’s implementations in line with
+the privacy and security recommendations in the specification, and is aligned
+with the overarching effort to deprecate powerful features on insecure origins.
+
+[Chrome Platform Status](https://www.chromestatus.com/feature/5688035094036480) &#124;
+[Chromium Bug](https://crbug.com/932078)
+
+
+## Remove Insecure Usage of DeviceOrientationEvent
```
Comment (kyarik) [inline src/content/en/updates/2019/06/chrome-76-deps-rems.md]:
```suggestion
## Remove Insecure Usage of `DeviceOrientationEvent`
```

**Comment by kyarik** [comment]
@googlebot rescan

**Comment by myakura** [comment]
although it's not a Blink web platform feature deprecation, I kinda want you to note that Flash is disabled by default in M76. I only see that deprecation in the enterprise notes. Isn't it worth notifying in the dev-related documentation as well?
https://support.google.com/chrome/a/answer/7679408?hl=en#75

**Pull request merged**